Source/ImPlotWidgets/Public/HistogramWidget.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) Coconut Lizard Limited. All rights reserved.
+#pragma once
+
+#include <Runtime/Core/Public/CoreMinimal.h>
+#include <c:/users/tia.ylinen/documents/unreal projects/vehiclegame/Plugins/UnrealImGui-ImPlot_Integration/Source/ImPlotWidgets/Public/ImPlotWidget.h>
```

Comment:
Need to path this include properly

Reply:
Here I had hoped I had noticed all of them just this once